app/Http/Controllers/IllustrationController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\EditIllustrationRequest;
+use App\Http\Requests\StoreIllustrationRequest;
+use App\Models\Illustration;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response as HttpResponse;
+
+class IllustrationController extends Controller
+{
+    public function illustrations(Request $request)
+    {
+        $id = $request->id;
+        $illustration = $id ? Illustration::findOrFail($id) : "";
```

Comment:
I replaced with the following code:  
```
$illustration = Illustration::find($request->id) ?? null;
```
Your suggestion (`$illustration = Illustration::findOrFail($id);`) does not work because of the method `findOrFail` that will always throw a 404 if the id does not exist and the execution of the code stops. Replacing `findOrFail` by `find `and assigning a fallback to `null `allows to continue the execution without throwing 404.

Reply:
Replaced with `$illustration = Illustration::find($request->id);`